### PR TITLE
Python Code 1 - Armstrong

### PR DESCRIPTION
**Pull request by Ganeshbabu18**

### CODE REVIEW

File: ArmstrongPy.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Python program to check if the number is an Armstrong number or not
+
 num = int(input("Enter a number: "))
 sum = 0
```
Comment (rohitlamsonge66) [inline ArmstrongPy.py]: Try to use method, function or lamda fuction for writing a program

Comment (Ganeshbabu18) [reply ArmstrongPy.py]: Kindly review the changes.... @amitsahu8 @rohitlamsonge66

File: ArmstrongPy.py [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
- sum += digit ** 3
- temp //= 10
-if num == sum:
- print(num,"is an Armstrong number")
-else:
- print(num,"is not an Armstrong number")
\ No newline at end of file
+class ArmStrong():
+ def Armstrong(self):
+    num = int(input("Enter a number: "))
```
Comment (amitsahu8) [inline ArmstrongPy.py]: Please pass the input directly, don't use the input to read it from console

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: ArmstrongPy.py (hunk 2 above)
Comment (Ganeshbabu18) [reply ArmstrongPy.py]: Noted and made the required changes, kindly verify  it

Comment (amitsahu8) [reply ArmstrongPy.py]: Resolve the conflicts

Comment (Ganeshbabu18) [reply ArmstrongPy.py]: @amitsahu8 Conflicts are resolved, Kindly check

File: ArmstrongPy.py [hunk 3]
```diff
@@ -1,13 +1,31 @@
...
+    temp = num
+    while temp > 0:
+        digit = temp % 10
+        sum += digit ** 3
+        temp //= 10
+    if num == sum:
+        print(num, "is an Armstrong number")
+    else:
+        print(num, "is not an Armstrong number")
+<<<<<<< HEAD
```
Comment (amitsahu8) [inline ArmstrongPy.py]: work on this HEAD

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: ArmstrongPy.py (hunk 3 above)
Comment (Ganeshbabu18) [reply ArmstrongPy.py]: @amitsahu8 Rework done, Kindly verify it

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**